### PR TITLE
Clarified OS access via API

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Comment by ymengms** [comment]
The changes for the FAQ page look good. I noticed this page also has similar statements https://docs.mulesoft.com/object-store/ 

Object Store v2 lets CloudHub applications store data and states across batch processes, Mule components and applications, **from within an application**. 

**On-premise Mule applications cannot use Object Store v2.**

is that something can be updated as well?

**Comment by hannanelson** [comment]
@ymengms - What do you want this to say? 

> Object Store v2 lets CloudHub applications store data and states across batch processes, Mule components, and applications, from within an application or by using the Object Store REST API.

> On-premise Mule applications can access Object Store v2 only using the Object Store REST API.

**Review by ymengms — APPROVED** [review]
The changes sound good to me. I think the object store team should have the final say.

Thank you!
Yuan

**Review by ymengms — DISMISSED** [review]

**Review by podviaznikov — DISMISSED** [review]
@hannanelson looks all good. Just one question. Do we still use `Cloudhub` name in the docs and not `Runtime Manager`? This is more general question about Anypoint platform terminology.

**Review by danieldatri — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -4,13 +4,20 @@ include::_attributes.adoc[]
...
 
-You can select Object Store v2 as the implementation for Mule 3 and Mule 4 in CloudHub by checking the *Object Store V2* checkbox in Runtime Manager at deployment time. See the xref:osv2-faq.adoc[FAQ] for Object Store information.
+You can select Object Store v2 as the implementation for Mule 3 and Mule 4 in CloudHub by checking the *Object Store V2* checkbox in Runtime Manager at deployment time.
+
+See the xref:osv2-faq.adoc[FAQ] for Object Store information.
 
 image::osv2-architecture.png[width=300]
 
-On-premise Mule applications cannot use Object Store v2.
+On-premise Mule applications can access Object Store v2 only using the Object Store REST API.
```
Comment (fermujica) [inline modules/ROOT/pages/index.adoc]: Shouldn't this be "On-premises Mule applications"?
Also 
```
can access Object Store v2 only using the Object Store REST API.
```
Would read better
```
can only access Object Store v2 by using the Object Store REST API.
```

**Review by fermujica — DISMISSED** [review]

**Review by fermujica — APPROVED** [review]

**Pull request merged**